api/admin/service.md

```diff
@@ -75,7 +75,7 @@ Now, calls to the X-Chain can be made to either `/ext/bc/X` or, equivalently, to
 
 Give a blockchain an alias, a different name that can be used any place the blockchain’s ID is used.
 
-:::note Aliasing a chain can also be done via the [Node API](https://docs.avax.network/nodes/configure/avalanchego-config-flags.md#--chain-aliases-file-string).
+:::note Aliasing a chain can also be done via the [Node API](/nodes/configure/avalanchego-config-flags.md#--chain-aliases-file-string).
```

Comment:
Should this be
```suggestion
:::note Aliasing a chain can also be done via the [Node API](/nodes/configure/avalanchego-config-flags#--chain-aliases-file-string).
```
?

Reply:
Nope sir, the ".md" should be kept when we use local path. you can see other examples in the service.md file. 
![image](https://github.com/ava-labs/avalanchego/assets/150209682/9a10b4d3-0c38-464f-9c58-4cc88978da2d)